<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Ubuntu::Internal::CreateTargetNamePage</class>
 <widget class="QWidget" name="Ubuntu::Internal::CreateTargetNamePage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>856</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QFormLayout" name="formLayout">
   <item row="0" column="0">
    <widget class="QLabel" name="label">
     <property name="text">
      <string>Target name: </string>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="Utils::FancyLineEdit" name="lineEditName"/>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>Utils::FancyLineEdit</class>
   <extends>QLineEdit</extends>
   <header location="global">utils/fancylineedit.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
